src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -32,53 +33,43 @@
 	private static Map<String, Integer> inventory = new HashMap<>();
 
 	private void checkout(List<Item> cart) {
+
 		Map<String, Integer> tempInventory = inventory;
 		for(Item item : cart) {
 			int currentInventory = tempInventory.get(item.getId());
 			if(currentInventory <= 0) {
 				String message = "No inventory for " + item.getId();
+				System.out.println(message);
 				throw new RuntimeException(message);
 			}
-
-			tempInventory.put(item.getId(), currentInventory--);
```

Comment:
this was not decrementing before being used by 'put'. Maybe it's a 'post' operator.

The previous [java-spring-boot-log4j2 repo had this as](https://github.com/sentry-demos/java-spring-boot-log4j/blob/master/src/main/java/io/sentry/example/Application.java#L45) `currentInventory-1` and not `currentInventory--`.

I couldn't get currentInventory-1 to work, so went with `currentInventory = currentInventory - 1;`